### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by farouqdaif1**
1-Created a database named vet_clinic.
2-Create a table named animals with the following columns:
  * id: integer
  * name: string
  * date_of_birth: date
  * escape_attempts: integer
  * neutered: boolean
  * weight_kg: decimal
3-Insert the given data in the table.
4-Wrote queries for the following:
  * Find all animals whose name ends in "mon".
  * List the name of all animals born between 2016 and 2019.
  * List the name of all animals that are neutered and have less than 3 escape attempts.
  * List the date of birth of all animals named either "Agumon" or "Pikachu".
  * List name and escape attempts of animals that weigh more than 10.5kg
  * Find all animals that are neutered.
  * Find all animals not named Gabumon.
  * Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
Copied all the SQL into the respective files in the generated repository.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+--   4 | Devimon | 2017-05-12    |               5 | t        |      11.0
+
+--List the name of all animals born between 2016 and 2019.
+
+select name from animals where date_of_birth between '2016-01-01'and '2019-12-31';
+--  Gabumon
+--  Devimon
+
+--List the name of all animals that are neutered and have less than 3 escape attempts.
+select name from animals where neutered= true and escape_attempts<=3;
```
Comment (mmsesay) [inline queries.sql]: - [ ] Just as the question states, you should list all animals that have less than 3 escape attempts. Your query is currently, listing animals who have less than 3 and those who have exactly 3 escaped attempts. And that shouldn't be the case.
Please remove the `=` symbol from your query.

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @farouqdaif1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (farouqdaif1) [reply queries.sql]: Thanks, I changed it.

**Review by t-yanick — APPROVED** [review]
Hi @farouqdaif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**